d20potz.py

```diff
@@ -282,7 +282,22 @@ async def hp_command(update: Update, context: ContextTypes.DEFAULT_TYPE):
 
     if len(params) > 1:       
         player_param = params[1].lower()
-        player_list = CONFIG.order.lower().split()    
+        player_list = CONFIG.order.lower().split()
+
+        if player_param == "all":
+            text=""
+            for player_name in player_list:
+                try:
+                    hp = get_player_hp(chat_id, player_name)
+                    player_text = '{} has {} hp\n'.format(spell_hero_name(player_name), hp)
+                except:
```

Comment:
Or there were some exception thrown which we happily ignored and did not even print to logs. Plz avoid using `except` like that. It's trivial to check if the key is in the DB or not.